docs/wiki/introduction/sql.md

```diff
@@ -12,6 +12,8 @@ The osquery SQL language is a superset of SQLite's, please read [SQL as understo
 
 `SELECT` only! All mutation-based verbs exist, like `INSERT`, `UPDATE`, `DELETE`, and `ALTER` but they do not do anything-- except if you're fancy and creating run-time tables or `VIEW`s. ;)
 
+> NOTICE: Several tables, `file` for example, requires a predicate for one of the columns, and **will not work without it**. See  [Tables with arguments](#tables-with-arguments) for more information.
```

Comment:
Sorry to nitpick, but "requires" should be "require", and there is an extra space after "See".